src/backend/Functions/FunctionsDIConfiguration.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using DependencyInjection;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Host.Config;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Fx.Portability.Azure.Storage;
+using Microsoft.Fx.Portability.ObjectModel;
+using Microsoft.WindowsAzure.Storage;
+using System;
+
+namespace Functions
+{
+    public class InjectConfiguration : IExtensionConfigProvider
+    {
+        public virtual void Initialize(ExtensionConfigContext context)
+        {
+            var services = new ServiceCollection();
+            RegisterServices(services);
```

Comment:
nit: new-line. See comment below

Reply:
Fixed

---
In reply to: [178092713](https://github.com/Microsoft/dotnet-apiport/pull/598#discussion_r178092713) [](ancestors = 178092713)